Added patch for libgag/src/FileManager.cpp in order to remove my own home
directory from first inspection by the included filemanager.  This can
trigger big syslog growth when automount is active.  Can have undesirable
effects in a system with automounter.
Debian bug #408733.

Index: libgag/src/FileManager.cpp
===================================================================
--- a/libgag/src/FileManager.cpp.orig	2008-03-08 06:27:16.000000000 +0100
+++ b/libgag/src/FileManager.cpp	2008-05-06 21:06:11.295638046 +0200
@@ -121,7 +121,7 @@
 
 			if ((linksize + 13) <= (int)sizeof(link))
 			{
-				strcat(link, "/share/glob2");
+				strcat(link, "/share/games/glob2");
 				addDir(link);
 			}
 		}
@@ -129,7 +129,7 @@
 		#endif
 
 		addDir(PACKAGE_DATA_DIR);
-		addDir(PACKAGE_SOURCE_DIR);
+		//addDir(PACKAGE_SOURCE_DIR);
 		fileListIndex = -1;
 	}
 	
